### PR TITLE
add graffitipot

### PR DESCRIPTION
**Pull request by p-chan**
This is 2ch browser application.

### CODE REVIEW

File: Casks/graffitipot.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'graffitipot' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://crystaly.com/graffitipot/GraffitiPot_1.1.zip'
+  name 'GraffitiPot'
+  homepage 'http://crystaly.com/graffitipot/'
+  license :freemium
```
Comment (vitorgalvao) [inline Casks/graffitipot.rb]: Is this a free app that works indefinitely but with limitations that can be removed by paying?

File: Casks/graffitipot.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'graffitipot' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/graffitipot.rb]: There’s clearly a version in the url. This needs to be set.

File: Casks/graffitipot.rb (hunk 1 above)
Comment (p1ch-jp) [inline Casks/graffitipot.rb]: No, This app offers all functions for free.

File: Casks/graffitipot.rb (hunk 2 above)
Comment (p1ch-jp) [inline Casks/graffitipot.rb]: Sorry, Added support.

File: Casks/graffitipot.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/graffitipot.rb]: Then it is `:gratis`, not `:freemium`.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 1ad240e4f2d517422c04bdd38e03121039b65104. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by p1ch-jp** [comment]
Thanks for your help.

**Pull request closed without merging**